README.md

```diff
@@ -155,19 +155,24 @@ lm_eval --model openai-completions \
-| Cohere                      | [:hourglass: - blocked on Cohere API bug](https://github.com/EleutherAI/lm-evaluation-harness/pull/395) | N/A                                                                            | [All `cohere.generate()` engines](https://docs.cohere.com/docs/models)                        | `generate_until`, `loglikelihood`, `loglikelihood_rolling` |
-| [Llama.cpp](https://github.com/ggerganov/llama.cpp) (via [llama-cpp-python](https://github.com/abetlen/llama-cpp-python))                        | :heavy_check_mark:              | `gguf`, `ggml`                                                                 | [All models supported by llama.cpp](https://github.com/ggerganov/llama.cpp)               | `generate_until`, `loglikelihood`, `loglikelihood_rolling` |
-| vLLM                        | :heavy_check_mark:       | `vllm`                                                                         | [Most HF Causal Language Models](https://docs.vllm.ai/en/latest/models/supported_models.html) | `generate_until`, `loglikelihood`, `loglikelihood_rolling`                             |
-| Your inference server here! | ...                             | ...                                                                            | ...                                                                                           | ...                                                      |                                | ...                                                      |
+| API or Inference Server                                                                                                   | Implemented?                    | `--model <xxx>` name                                             | Models supported:                                                                             | Request Types:                                             |
+|---------------------------------------------------------------------------------------------------------------------------|---------------------------------|------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|------------------------------------------------------------|
+| OpenAI Completions                                                                                                        | :heavy_check_mark:              | `openai-completions`                                             | up to `code-davinci-002`                                                                      | `generate_until`, `loglikelihood`, `loglikelihood_rolling` |
```

Comment:
could we add the `openai-chat-completions`, `local-chat-completions`, `local-completions`, `openai-completions` all to this table as model names that one can call?

Reply:
do we need to register `local-completions` as `@register_model` in the class?

Reply:
yup, let's not add `local-completions` yet until the base_url + HF tokenizer options are supported though.

Reply:
Ok, let me know if I need to change anything else here

Reply:
If you give me edit access on your PR branch I can make these changes to not waste your time on minor formatting stuff, but these are the last ones (I'm sorry!)

- OpenAI Completions model name should become -> `openai-completions`
- OpenAI ChatCompletions model name should go from `N/A` -> `openai-chat-completions, local-chat-completions`
- Your Inference Server -> " `local-chat-completions` (using `openai-chat-completions` model type)" | `Any server address that accepts GET requests using HF models and mirror's OpenAI's ChatCompletions interface`

Reply:
Thank you! Ok I think I fixed it - one last check please?

Reply:
(I was also looking for this option but don't see it in the PR :\ https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
<img width="397" alt="Screenshot 2023-12-20 at 3 37 37 PM" src="https://github.com/EleutherAI/lm-evaluation-harness/assets/3837836/21009dad-b096-422d-9e1d-f54bd74595bd">

Reply:
hm, there's usually an option in the sidebar you can check that says something like "allow maintainers to edit this PR". Oh well. Thank you for bearing with me, LGTM now!